package.json

```diff
@@ -7,7 +7,7 @@
     "lint": "eslint src",
     "test": "jest",
     "dev": "tsc -w --declaration",
-    "build": "rm -rf dist && tsc --declaration; cp -r ./src/types ./dist/types",
+    "build": "rm -rf dist && tsc --declaration",
```

Comment:
In general it's not good idea to use such complex commands. For example you can't run them on windows, because there is no `cp` and `&&` there.

It's better to use something like gulp.

If you want to keep it simple you can use two common packages:
https://www.npmjs.com/package/shx
https://www.npmjs.com/package/npm-run-all

And rewrite this line like this:
```
"build": "npm-run-all build:*",
"build:rm": "shx rm -rf dist",
"build:tsc": "tsc --declaration",
```

But gulp is faster.